train_od_ex.py

```diff
@@ -0,0 +1,288 @@
+import os
+import random
+import numpy as np
+
+import matplotlib
+import matplotlib.pyplot as plt
+import tensorflow as tf
+
+from object_detection.utils import config_util
+from object_detection.builders import model_builder
+
+from utils import load_image_into_numpy_array
+from utils import plot_detections
+
+OD_MODELS_PATH = '.'
+os.chdir(OD_MODELS_PATH)
+# For interactive plot in PyCharm
+matplotlib.use('module://backend_interagg')
+
+
+class RubberDucksTraining:
```

Comment:
Why do we need DuckTraining at all? How this related to our project?

Reply:
Please, change names and structure more related to our project w/o some rubber ducks

Reply:
Rubber ducks are quite good as simplest validation dataset